### PR TITLE
Feature/add klaos disclamer

### PR DESCRIPTION
**Pull request by zorjak**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -112,6 +113,20 @@ func run() error {
 		if err != nil {
 			return fmt.Errorf("error instantiating eth client: %w", err)
 		}
+
+		chainID, err := client.ChainID(ctx)
+		if err != nil {
+			return err
+		}
+		slog.Debug("zoran debug", "chainID", chainID.String())
```
Comment (kudlaj) [inline cmd/main.go]: zoran debug 😄

**Review by kudlaj — CHANGES_REQUESTED** [review]

File: cmd/main.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ import (
 var version = "undefined"
 
 const historyLength = 256
+const KLAOSChainID = 2718
```
Comment (luispdm) [inline cmd/main.go]: Lowercase, please. No need to export this constant

**Review by luispdm — CHANGES_REQUESTED** [review]

File: cmd/main.go (hunk 1 above)
Comment (zorjak) [reply cmd/main.go]: delted

File: cmd/main.go (hunk 2 above)
Comment (zorjak) [reply cmd/main.go]: changed

**Review by kudlaj — APPROVED** [review]

**Review by luispdm — APPROVED** [review]

**Pull request merged**